mission_control/test/load_execute_mission.py

```diff
@@ -44,38 +44,22 @@
 from mission_control.msg import ReportExecuteMissionState
 from mission_control.msg import ReportLoadMissionState
 from mission_control.msg import ReportMissions
-from mission_control.msg import FixedRudder
 
 
-def wait_for(predicate, period=1):
-    while not rospy.is_shutdown():
-        result = predicate()
-        if result:
-            return result
-        rospy.sleep(period)
-    return predicate()
-
-
-class TestFixedRudderBehavior(unittest.TestCase):
-    """ 
-        The test loads and execute a fixed rudder mission.
+class LoadExecuteMission:
```

Comment:
@patapufate nit: perhaps `MissionInterface` is a better name ? `LoadExecuteMission` suggests an action.

Reply:
fixed!